lib/chore/queues/sqs/consumer.rb

```diff
@@ -114,11 +114,12 @@ def queue_timeout
 
         # Access to the configured SQS connection object
         def sqs
-          @sqs ||= AWS::SQS.new(
+          sqs_options = {
             :access_key_id => Chore.config.aws_access_key,
-            :secret_access_key => Chore.config.aws_secret_key,
-            :logger => Chore.logger,
-            :log_level => :debug)
+            :secret_access_key => Chore.config.aws_secret_key
+          }
+          sqs_options.merge!(:logger => Chore.logger, :log_level => :debug) if Chore.config.log_level == Logger::DEBUG
```

Comment:
Would it not be best to keep having it use the Chore.logger, and simply default it's internal log level to match what chore is configured to do?